source/shared/core_stmt.cpp

```diff
@@ -1631,7 +1140,8 @@ void stmt_option_query_timeout:: operator()( _Inout_ sqlsrv_stmt* stmt, stmt_opt
 
 void stmt_option_send_at_exec:: operator()( _Inout_ sqlsrv_stmt* stmt, stmt_option const* /*opt*/, _In_ zval* value_z )
 {
-    core_sqlsrv_set_send_at_exec( stmt, value_z );
+    // zend_is_true does not fail. It either returns true or false.
+    stmt->send_streams_at_exec = (zend_is_true(value_z)) ? true : false;
```

Comment:
Perhaps 'stmt->send_streams_at_exec = zend_is_true(value_z)' is enough?  I think the zend_is_true function is understandable by itself when reading through the code.